### ISSUE TITLE
examin is broken with new defaults

### ISSUE THREAD

**Issue by dworogers** [nrc-cnrc/EGSnrc#372]
If I use examin for electron data only, the user is never asked about photon options and yet I get this message:
```
You must turn ON Compton binding corrections when using
a detailed atomic relaxation (eadl_relax=true) since 
binding energies taken from incoh.data below 1 keV!
```
So I asked for e- and photon data and turned on bound compton when asked, but then I get the message
```
   Scattering off bound electrons creates atomic vacancies,
   potentially starting an atomic relaxation cascade. 
   Please turn ON atomic relaxations.
```
But the user is never asked about atomic relaxations. For the purposes of examin they are irrelevant, but this trap halts the code anyway.

**Comment by dworogers** [comment]
Hi Reid,

Here is my output. I tried it with different data sets and materials and
get the same problem.


97_atlas03>examin -p 700icru
================================================================================
EGSnrc version 4 for x86_64-unknown-linux-gnu          Tue Nov  7 13:09:43
2017
================================================================================
configuration........................................ebAug22
user code............................................examin
pegs file............................................700icru on HEN_HOUSE
using
host...........................................atlas03.physics.carleton.ca
output file(s).......................................test
================================================================================
  Media available in this data file are:
  MEDIUM=AG700ICRU               ,STERNCID=AG700ICRU
  MEDIUM=H2O700ICRU              ,STERNCID=H2O700ICRU
  Material identifier: H2O700ICRU


  Title after  H2O700ICRU
  Electrons and photons(0), only electrons(1) or photons(2): 1
  Tables(0) or individual energies(1):
  Output  terminal(0),  disk file(1), terminal&plot(2), disk&plot(3): 1

  Call HATCH
  ===> Photonuclear flag:   0
  Rayleigh data available for medium  1 in PEGS4 data set.

(Re)-initializing photon cross sections with files from the series: xcom

  Compton cross sections: default
  Using Compton cross sections from
/home/drogers/ebAug22/EGSnrc_with_egs_brachy/HEN_HOUSE/data/compton_sigma.data
  Working on medium   1 ... OK

Reading screened Rutherford MS data ...............  done

   Medium            1  sige =   0.43643562363412508
0.42997804778658849       monotone =  T T

   Initializing tmxs for estepe =   0.25000000000000000       and ximax = 0.50000000000000000


***************** Error: 
You must turn ON Compton binding corrections when using
a detailed atomic relaxation (eadl_relax=true) since 
binding energies taken from incoh.data below 1 keV!

***************** Quiting now.
================================================================================





My install was from Aug 22 using the egs_brachy develop branch

I cannot tell from the examin.mortran file anymore when it was last
changed. I am sure that github experts can figure it out, but I can't :-(

Thanks for looking at this

Dave

**Comment by rtownson** [comment]
This was fixed in PR #337, and the bug was only present on the `develop` branch.
